app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/CreateDLCOfferDialog.scala

```diff
@@ -0,0 +1,436 @@
+package org.bitcoins.gui.dlc.dialog
+
+import org.bitcoins.cli.CliCommand.CreateDLCOffer
+import org.bitcoins.core.currency.{Bitcoins, Satoshis}
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.dlc._
+import org.bitcoins.core.protocol.tlv._
+import org.bitcoins.core.wallet.fee.SatoshisPerVirtualByte
+import org.bitcoins.gui.GlobalData
+import org.bitcoins.gui.dlc.DLCPlotUtil
+import org.bitcoins.gui.util.GUIUtil.{numberFormatter, setNumericInput}
+import scalafx.Includes._
+import scalafx.geometry.{Insets, Pos}
+import scalafx.scene.control._
+import scalafx.scene.layout._
+import scalafx.stage.Window
+
+import java.time.ZoneOffset
+import scala.collection._
+import scala.util.{Failure, Success, Try}
+
+object CreateDLCOfferDialog {
+
+  def showAndWait(parentWindow: Window): Option[CreateDLCOffer] = {
```

Comment:
This method is 400 lines. We need to break this up and move all the helper methods out of it.

Reply:
Did some simplifications, however a lot of this requires the elements in the gui